### PR TITLE
Consolidate surefire systemPropertyVariables in ITs

### PR DESCRIPTION
**Pull request by famod**
removed entries are already set in build-parent

See also: #9523

### CODE REVIEW

**Review by gsmet — APPROVED** [review]

File: integration-tests/scala/pom.xml [hunk 1]
```diff
@@ -73,7 +73,6 @@
                 <configuration>
                     <systemPropertyVariables>
                         <!--suppress MavenModelInspection -->
-                        <maven.home>${maven.home}</maven.home>
                         <maven.repo>${settings.localRepository}</maven.repo>
```
Comment (gsmet) [inline integration-tests/scala/pom.xml:76]: I wonder if we should move this one to the build-parent too. @aloubyansky WDYT?

Comment (famod) [reply integration-tests/scala/pom.xml:76]: Shouldn't hurt!

/cc @aloubyansky

**Comment by famod** [comment]
Btw, another potential consolidation (by pulling up into `build-parent`) could be this for `maven-failsafe-plugin`:
```
<systemPropertyVariables>
  <native.image.path>${project.build.directory}/${project.build.finalName}-runner</native.image.path> 
</systemPropertyVariables>
```

**Comment by gsmet** [comment]
I think it would be better to have it in the `integration-tests` parent. We have one, let's use it.

**Comment by famod** [comment]
Yeah, even better. I'd have a look in a separate PR.

File: integration-tests/scala/pom.xml (hunk 1 above)
Comment (aloubyansky) [reply integration-tests/scala/pom.xml:76]: Agreed.

Comment (famod) [reply integration-tests/scala/pom.xml]: Done, also for `failsafe`.

I see no harm in adding this:
```xml
<project.version>${project.version}</project.version>
```
to build-parent (or IT parent) as well, but I did not touch it for now.

**Comment by famod** [comment]
CI passed in my fork: https://github.com/famod/quarkus/actions/runs/397981678

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**